packages/elixir.rb

```diff
@@ -8,16 +8,8 @@ class Elixir < Package
   source_sha256 '70972b844c12bc1a3960136d628ab4f21ca87dd5539c544ebabe41d6c9239ba9'
```

Comment:
Wasn't there an update available for this package?

Reply:
OK, I know what the issue is here.  This package was already updated in #1665.  You can remove this package from the PR.